src/main/java/org/myrobotlab/framework/MrlListenerComparator.java

```diff
@@ -0,0 +1,20 @@
+package org.myrobotlab.framework;
+
+import java.util.Comparator;
+
+import org.myrobotlab.service.config.ServiceConfig.Listener;
+
+/**
+ * 
+ * @author GroG
```

Comment:
`@author` should be at the bottom so it renders correctly for all javadoc parsers

Reply:
:+1: